src/routes/onboarding/set-medication-reminders/medication-section.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import { Form, Button } from "../../../components"
+import { faTrashCan } from "@fortawesome/free-regular-svg-icons";
+import { MedicationTimeInput } from "./medication-time-input";
+
+interface Props {
+  index?: number;
+  onDelete?: (index:number) => void;
+  elementId?: number;
+}
+
+export const MedicationSection: React.FC<Props> = ({ index, onDelete, elementId}) =>  {
+
+  const {
+    register,
+    unregister,
+    formState: {errors},
+  } = useForm();
+
+  const inputName = `medication${elementId}` || "medication6"
```

Comment:
a bit confused about the fallback to "medication6" but maybe I am missing something 😅